semantics.md

```diff
@@ -2514,8 +2525,10 @@ deviates from MongoDB\'s comparison semantics.
 
 The binary comparison operators \<, \<=, \<\>, =, \>, and \>= specify
 less than, less than or equals, not equals, equals, greater than, and
-greater than or equals, respectively. MongoSQL does not support
-comparison operations on structured data (documents and arrays).
+greater than or equals, respectively. Document comparison follows MongoDB's 
+equality semantics, requiring exact field name and value matches, with 
+field order being significant. Array comparison requires equal length 
+and equal elements in the same order.
```

Comment:
In the Rejected Alternatives for this section (roughly lines 2750 - 2755) we note that "For now, we rejected comparisons for structured data (documents and arrays)". Can you update that section to say "_Initially_, we rejected comparisons for structured data ..." (and any other little bits of clean up in that paragraph to indicate we previously rejected this but no longer do).

Reply:
Oh good catch, I've updated it.